README.md

```diff
@@ -11,6 +11,10 @@ Seriously, we need and want the help. Want to learn how to code? Or maybe you kn
 autoscend is a script that will play through an entire ascension for you in the Kingdom of Loathing.
 It is built up from sl\_ascend and cc\_ascend before it.
 
+autoscends goal is supposed to be able to brute force an ascension for all paths of KoL, Not to do it optimally, but within a couple real-life days of target. If you bear this in mind while you use it you will have a much better experience.  
```

Comment:
Autoscend's goal is to be able to brute force an ascension for all paths of KoL, not to do it optimally, but within a couple of real-life days of the target. If you bear this in mind while you use it, you will have a much better experience.